code/modules/jobs/job_types/internal_affairs_agent.dm

```diff
@@ -0,0 +1,62 @@
+/datum/job/iaa
+	title = "Internal Affairs Agent"
+	flag = IAA
+	department_head = list("Captain")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "Central Command"
+	selection_color = "#ddddff"
+	chat_color = "#50C878"
+	exp_requirements = 840
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/iaa
+
+	access = list(ACCESS_IAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO)
+	minimal_access = list(ACCESS_IAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO)
```

Comment:
no

Reply:
This is basic access only and is intended to be used for departmental inspections. Ideally the CAA will contact the department's head of staff or a staff member to be allowed further into the department for inspections.

Reply:
removed this access